core/src/main/java/io/grpc/Metadata.java

```diff
@@ -177,6 +178,14 @@ public T apply(MetadataEntry entry) {
   }
 
   /**
+   * Returns set of all keys in store
```

Comment:
Looks like check style wants this to be a sentence with a period.  Maybe change it to:

/**
    * Returns set of all keys in store.
    *
    * @return ImmutableSet of keys
    */